### PR TITLE
apachectl: add page

### PR DESCRIPTION
**Pull request by basiclines**

### CODE REVIEW

**Comment by agnivade** [comment]
A page should be in the platform specific folder. In this case, it should be inside the osx folder. This instruction was explicitly mentioned in the PR template.

Please move this page under the osx folder.

**Comment by basiclines** [comment]
@agnivade Just moved the page to the osx folder 😅

File: pages/osx/apachectl.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# apachectl
+
+> Apache HTTP Server Control Interface for OSX.
```
Comment (sbrl) [inline pages/osx/apachectl.md]: I'm not sure that `Control Interface` needs to be in title case. Perhaps lowercase would read better here?

Comment (basiclines) [reply pages/osx/apachectl.md]: Fixed!

File: pages/osx/apachectl.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# apachectl
+
+> Apache HTTP Server control interface for macOS.
+
+- Load the org.apache.httpd launchd job:
```
Comment (agnivade) [inline pages/osx/apachectl.md]: I am a bit curious about why the words "Load" and "Unload" are used and not "start" and "stop".

File: pages/osx/apachectl.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`apachectl start`
+
+- Unload the org.apache.httpd launchd job:
+
+`apachectl stop`
+
+- Unload, then loads the org.apache.httpd launchd job:
```
Comment (agnivade) [inline pages/osx/apachectl.md]: You don't have to mention "org.apache.httpd" after the 2nd time. It is implicit.

File: pages/osx/apachectl.md (hunk 2 above)
Comment (basiclines) [reply pages/osx/apachectl.md]: That's from the official `man` page but it's true that it doesn't fit the needs of this repo.

Comment (agnivade) [reply pages/osx/apachectl.md]: Yes, we try to avoid copying text verbatim from the man page. And try to change it to something simple and clear.

**Pull request merged**